### PR TITLE
Mrizzo fix spaces in trader demo

### PR DESCRIPTION
**Pull request by mat-rizzo-r3**
https://r3-cev.atlassian.net/browse/CORDA-728

Everything is quoted, and now works on Mac. Tested trader demo and bank of corda demo

### CODE REVIEW

File: gradle-plugins/cordformation/src/main/kotlin/net/corda/plugins/Cordformation.kt [hunk 1]
```diff
@@ -18,10 +18,18 @@ class Cordformation : Plugin<Project> {
          * @return A file handle to the file in the JAR.
          */
         fun getPluginFile(project: Project, filePathInJar: String): File {
-            val archive: File? = project.rootProject.buildscript.configurations.single { it.name == "classpath" }.find {
-                it.name.contains("cordformation")
-            }
-            return project.rootProject.resources.text.fromArchiveEntry(archive, filePathInJar).asFile()
+            val archive: File? = project
```
Comment (Clintonio) [inline gradle-plugins/cordformation/src/main/kotlin/net/corda/plugins/Cordformation.kt]: Can you keep the getters inline? The lines with the lambdas on are fine to break out though.

File: gradle-plugins/cordformation/src/main/kotlin/net/corda/plugins/Cordformation.kt [hunk 2]
```diff
@@ -18,10 +18,18 @@ class Cordformation : Plugin<Project> {
...
+                    .rootProject
+                    .buildscript
+                    .configurations
+                    .single { it.name == "classpath" }
+                    .find { it.name.contains("cordformation") }
+            return project
```
Comment (Clintonio) [inline gradle-plugins/cordformation/src/main/kotlin/net/corda/plugins/Cordformation.kt]: Same here. Getters tend to be inline but the function calls would be fine to break out.

File: gradle-plugins/cordformation/src/main/kotlin/net/corda/plugins/Node.kt [hunk 3]
```diff
@@ -128,7 +128,7 @@ class Node(private val project: Project) : CordformNode() {
             // Can't parse as an X500 name, use the full string
             name
         }
-        nodeDir = File(rootDir.toFile(), dirName.replace("\\s", ""))
+        nodeDir = File(rootDir.toFile(), dirName)
```
Comment (Clintonio) [inline gradle-plugins/cordformation/src/main/kotlin/net/corda/plugins/Node.kt]: That explains a lot.

File: gradle-plugins/cordformation/src/noderunner/kotlin/net/corda/plugins/NodeRunner.kt [hunk 4]
```diff
@@ -104,7 +104,7 @@ private class TerminalWindowJavaCommand(jarName: String, dir: File, debugPort: I
     delay 0.5
     tell app "System Events" to tell process "Terminal" to keystroke "t" using command down
     delay 0.5
-    do script "bash -c 'cd $dir; ${command.joinToString(" ")} && exit'" in selected tab of the front window
+    do script "bash -c 'cd \"$dir\" ; \"${command.joinToString("""\" \"""")}\" && exit'" in selected tab of the front window
```
Comment (Clintonio) [inline gradle-plugins/cordformation/src/noderunner/kotlin/net/corda/plugins/NodeRunner.kt]: Top tier quote nesting.

File: core/src/main/kotlin/net/corda/core/schemas/NodeInfoSchema.kt [hunk 5]
```diff
@@ -85,13 +85,12 @@ object NodeInfoSchemaV1 : MappedSchema(
...
     data class DBPartyAndCertificate(
             @Id
-            @Column(name = "owning_key", length = 65535, nullable = false)
-            val owningKey: String,
-
-            //@Id // TODO Do we assume that names are unique? Note: We can't have it as Id, because our toString on X500 is inconsistent.
             @Column(name = "party_name", nullable = false)
             val name: String,
 
+            @Column(name = "owning_key", length = 65535, nullable = false)
```
Comment (ragmondo) [inline core/src/main/kotlin/net/corda/core/schemas/NodeInfoSchema.kt]: A non-nullable column of 65k width ? It can't any thinner?

**Pull request merged**